doc/spec.md

```diff
@@ -3755,6 +3768,68 @@ y = set([2, 3])
 x.union(y)                              # set([1, 2, 3])
 ```
 
+<a id='set·add'></a>
+### set·add
+
+`S.add(x)` adds the value `x` to the set `S`. This has no effect if `x` is already in `S`.
+
+```python
+x = set([1, 2])
+x.add(3)                             # None (x == set([1, 2, 3]))
+x.add(3)                             # None (x == set([1, 2, 3]))
+```
+
+<a id='set·pop'></a>
+### set·pop
+
+`S.pop()` removes an arbitrary item from the set and returns it.
```

Comment:
There is no "arbitrary" in Starlark, so let's describe this as the "first inserted element".